code/modules/vore/eating/vorepanel_vr.dm

```diff
@@ -304,7 +304,7 @@
 			return set_attr(usr, params)
 
 		if("saveprefs")
-			if(!ishuman(host) && !issilicon(host))
+			if(host.real_name != host.client.prefs.real_name || !ishuman(host) && !issilicon(host))
```

Comment:
```suggestion
			if(host.real_name != host.client.prefs.real_name || (!ishuman(host) && !issilicon(host)))
```

Just adding parens around the latter part. I think it would be fine without (&& binds stronger than || according to http://www.byond.com/docs/ref/#/operator) but still.